examples/opentelemetry/envoy-1.yaml

```diff
@@ -0,0 +1,132 @@
+# This proxy listens on 2 ports:
+#
+#   10000 -> routes to `service-1`
+#   10001 -> routes to `envoy-2`
+
+node:
+  # The cluster name is used by OpenTelemetry to identify the proxy.
+  cluster: envoy-proxy-1
```

Comment:
is this correct - i think the id is from the `service-name` below - probably lets remove this and add a comment about the service name (unless im wrong)

Reply:
make sense, thank you